mne/io/base.py

```diff
@@ -684,63 +685,36 @@ def set_annotations(self, annotations, emit_warning=True):
             Annotations to set.
         emit_warning : bool
             Whether to emit warnings when limiting or omitting annotations.
+
+        Returns
+        -------
+        self : instance of Raw
+            The raw object with annotations.
         """
-        new_annot = None
-        if annotations is not None:
-            if not isinstance(annotations, Annotations):
-                raise ValueError('Annotations must be an instance of '
-                                 'mne.Annotations. Got %s.' % annotations)
-
-            new_annot = annotations.copy()
-            meas_date = _handle_meas_date(self.info['meas_date'])
-
-            if new_annot.orig_time is not None:
-                offset = (new_annot.orig_time - meas_date -
-                          self.first_samp / self.info['sfreq'])
-            else:
-                offset = 0
-
-            omit_ind = list()
-            omitted = limited = 0
-            for ind, onset in enumerate(new_annot.onset):
-                onset += offset
-                if onset > self.times[-1]:
-                    omitted += 1
-                    omit_ind.append(ind)
-                    logger.debug('Omitting %d @ %s > %s'
-                                 % (ind, onset, self.times[-1]))
-                elif onset < self.times[0]:
-                    if onset + new_annot.duration[ind] < self.times[0]:
-                        omitted += 1
-                        omit_ind.append(ind)
-                        logger.debug('Omitting %d @ %s < %s'
```

Comment:
All the logger and emit_warning has not been implemented and the tests still pass.